test/requestContextPlugin.spec.js

```diff
@@ -1,14 +1,17 @@
 const fastify = require('fastify')
+const middie = require('middie')
 const { fastifyRequestContextPlugin } = require('../lib/requestContextPlugin')
 const { TestService } = require('./internal/testService')
 
 function initApp(endpoint) {
   const app = fastify({ logger: true })
-  app.register(fastifyRequestContextPlugin)
+  return app.register(middie).then(() => {
+    app.register(fastifyRequestContextPlugin)
 
-  app.get('/', endpoint)
+    app.get('/', endpoint)
 
-  return app.ready()
+    return app.ready().then(() => app)
```

Comment:
why this change? I think this is fixed `ready()` resolves to `app`.

Reply:
Was the fix released? it was returning undefined for me before I did the change.

Reply:
Updated to 3.0.3, that resolved it.